docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,104 @@
-# Why Use GTFS?
+# Reach more riders and improve public transit with GTFS
+GTFS, the open-source standard for transit data, empowers you to make public transportation more accessible and efficient.
 
-Over 10,000 transit agencies in 100+ countries rely on GTFS, quickly becoming an industry standard. Here's what makes it so advantageous:
+GTFS serves as the backbone of modern transit systems, enabling agencies to provide accurate and up-to-date information to riders, app developers, and policymakers. Below, we'll explore the various benefits of GTFS and how it enhances public transit worldwide.
 
-- **Simple**: The core GTFS data is text-based and easy to understand. 
-- **Flexible**: It also offers the possibility of plugging in additional features beyond schedules, like fare information, flexible services, and accessibility information.
-- **Open Source**: by being freely available, data can be shared easily, and developers can create tools, shaping the future of GTFS to better suit their needs.
-- **Community Driven**: developed collaboratively, GTFS ensures the format meets the evolving needs of riders and agencies.
+## GTFS: More Than Just Trip Planning
```

Comment:
I think this is a bit confusing since all the features we're listing below are actually all about trip planning. Maybe we're confusing this section with the one where we usually highlight other applications of GTFS, for example for research. A more appropiate title to keep the content can be "GTFS: More Than Just Schedules" since we're listing features that go beyond schedules (we can also go back to "GTFS can probably do more than you think")

Reply:
Yeah I can stand behind that title

Reply:
+1 for "GTFS can probably do more than you think"